### PR TITLE
docs(trace): added assumptions and non-functional requirements to architecture

### PR DESCRIPTION
**Pull request by markuskeidl**

### CODE REVIEW

File: docs-kits/kits/Traceability Kit/page_architecture-view.md [hunk 1]
```diff
@@ -12,6 +12,19 @@ Architecture View of the Kit.
 
 The following page offers an architecture perspective including the main building blocks and information regarding communication between different components, shown as sequence diagrams in a runtime view.
 
+In general, data must be provided to the Catena-X network using the Data Space Protocol (DSP). As standard for digital twins the Asset Administration Shell (AAS) standard is used - this is relevant for registering digital twins (in the Digital Twin Registry) as well as for providing digital twin data. SAMM is used as modelling language to describe the data of digital twins as aspects models.
```
Comment (mhellmeier) [inline docs-kits/kits/Traceability Kit/page_architecture-view.md:15]: Maybe we should link the Data Space Protocol (DSP) here?

**Review by mhellmeier — CHANGES_REQUESTED** [review]

File: docs-kits/kits/Traceability Kit/page_architecture-view.md (hunk 1 above)
Comment (markuskeidl) [reply docs-kits/kits/Traceability Kit/page_architecture-view.md:15]: I have no official link to this protocol. We also don't have a link to the AAS standard - probably in other places as well. I might make most sense to either add this to any reference of DSP or AAS or ... or to add a reference section somewhere.

**Review by mhellmeier — APPROVED** [review]
LGTM

File: docs-kits/kits/Traceability Kit/page_architecture-view.md (hunk 1 above)
Comment (matgnt) [reply docs-kits/kits/Traceability Kit/page_architecture-view.md:15]: DSP Link would be: https://docs.internationaldataspaces.org/dataspace-protocol/

Comment (matgnt) [inline docs-kits/kits/Traceability Kit/page_architecture-view.md:15]: should we reference SAMM as well?

**Review by matgnt — APPROVED** [review]

**Pull request merged**